### PR TITLE
upgrade base image packages while building images

### PR DESCRIPTION
**Pull request by phoracek**
Images we use have known issues in their packages, make sure to upgrade
on build.

### CODE REVIEW

**Comment by phoracek** [comment]
@booxter @SchSeba

File: build/registry/Dockerfile [hunk 1]
```diff
@@ -2,6 +2,8 @@ FROM quay.io/openshift/origin-operator-registry
 
 COPY manifests /registry
 
+RUN yum -y update
```
Comment (SchSeba) [inline build/registry/Dockerfile]: Oh you can't run this command on this image (this user there doesn't have permissions) I think we can remove the `yum update` from this image

Comment (phoracek) [reply build/registry/Dockerfile]: quay reports that it found vulnerabilities there, lemme try to use sudo

Comment (phoracek) [reply build/registry/Dockerfile]: `sudo` is not installed in the image. dropped update of the registry image for now.

Comment (SchSeba) [reply build/registry/Dockerfile]: I think the pod doesn't have the sudo command. you can try to save the user then run `USER root` yum update and return the USER again.

I am not sure if this going to work but you can try

**Pull request merged**